### ISSUE TITLE
Is the 1.12 version meant to work with Railcraft?

### ISSUE THREAD

**Issue by AlexJ6301** [MineMaarten/Signals#87]
Testing with an alpha of railcraft 1.12 and signals don't seem to work at all.
They work sometimes, but for two carts on the same loop, at least one doen't trigger the signals.
Manually set signals ignore all carts.
Shift sneak doesn't reset things.
etc etc
And, are locos meant to be detected?

**Comment by MineMaarten** [comment]
'an alpha', as far as I can see there are no public releases for Railcraft for MC 1.12. When this is the case I can look into this. Feel free to notify me again when this is the case.

Signals is designed to be friendly with modded rails/carts, so it should just work for the most part. I don't mind testing myself for the things you describe, but I'm going to wait for an official build.

"And, are locos meant to be detected?"
No special code has been written to support locomotives, and you mean stopping them at signals right? I would like to add support though.

**Comment by MineMaarten** [comment]
Yes, Railcraft Switch support was also something to look into (see https://github.com/MineMaarten/Signals/issues/26 ), I was just waiting for the Signals and RC targeted MC versions to align, which turns out to be tricky ;).

**Comment by liach** [comment]
i will help on this issue.

**Comment by liach** [comment]
@AlexJ6301 For a loop, how many signals did you put? one is useless; only two can create two blocks in which one is free.

**Comment by AlexJ6301** [comment]
Hi Liach. Yes, I tried loops with multiple signals in a loop.  They only pick up one of the carts.  I think Maarten is right to wait for a 1.12 RC release and see what happens then.
